src/components/Calendar/Calendar.tsx

```diff
@@ -0,0 +1,78 @@
+import React, { useState, ChangeEvent, useEffect } from "react";
+import { getCurrentWeek } from "../../utils/dateUtils";
+import MealInputCell from "../MealInputCell/MealInputCell";
+import { mealTypes } from "../../types/types";
+import "./styles.scss";
+
+const Calendar: React.FC = () => {
+  const currentWeekDays = getCurrentWeek();
+
+  const [inputs, setInputs] = useState<{ [key: string]: string }>(() => {
+    const savedInputs = localStorage.getItem("calendarInputs");
+    if (savedInputs) {
+      return JSON.parse(savedInputs);
+    }
+    return Object.fromEntries(
+      currentWeekDays.flatMap((date) =>
+        mealTypes.map((meal) => [`${date.toDateString()}-${meal}`, ""])
```

Comment:
do u plan to use `date` in another way which is not `date.toDateString()`?
if not, you should consider changing it to this in the original location because I see you are using it 5 times here